jenkins/dockerfiles/Dockerfile.builder_base

```diff
@@ -0,0 +1,7 @@
+FROM debian:buster
+MAINTAINER "Matt Hart" <matt@mattface.org>
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install bc bsdmainutils make bison flex ccache wget libelf-dev tar python2.7 python-requests python-pyelftools libssl-dev lzop
+
+#whoami doesnt work in docker
```

Comment:
dumb Q: Who/what is depending on `whoami`?

Reply:
@khilman Not certain, but *something* in the kernel builds uses it, because you get output like this without it
http://staging-storage.kernelci.org/mattface/master/v4.15-rc8-43271-g452b16a33fcd/x86/allnoconfig/build.log